devices/philips.js

```diff
@@ -2586,8 +2586,8 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}),
     },
     {
-        zigbeeModel: ['LWE003'],
-        model: '9290020400',
+        zigbeeModel: ['LWE003', 'LWE008'],
+        model: '9290020400/929003021301',
```

Comment:
If I understand correctly, LWE008 = 929003021301, meaning we can distinguish these models. If yes, then please duplicate the definition.